opennms-webapp-rest/src/main/webapp/WEB-INF/applicationContext-cxf-rest-v2.xml

```diff
@@ -44,6 +44,7 @@
     </bean>
     <bean id="openApiFeature" class="org.apache.cxf.jaxrs.openapi.OpenApiFeature">
         <property name="title" value="OpenNMS V2 RESTful API" />
+        <property name="version" value="1.0.0" />
```

Comment:
Should this be independent of Horizon/Meridian version ?  
If that is standard practice, we can start with minor version like `0.1.0` since we currently don't support OpenAPI for all the Rest API

Reply:
I'm fine with any version number we decide to start with here.

Reply:
I've updated the version number to 0.1.0